src/commands/autocomplete/path.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import AutocompleteBase from '.'
+import path from 'path'
+
+export default class AutocompletePath extends AutocompleteBase {
+  static topic = 'autocomplete'
+  static command = 'commandspath'
+  static description = 'path to autocomplete commands'
+  static hidden = true
+
+  async run () {
```

Comment:
Thought about it, but decided if I have to call just 1 command, I'd rather it be explicit about my intentions then user super or whatnot.